subsys/bluetooth/shell/bt.c

```diff
@@ -1244,6 +1244,10 @@ static int cmd_hci_cmd(const struct shell *sh, size_t argc, char *argv[])
 		}
 
 		buf = bt_hci_cmd_create(BT_OP(ogf, ocf), len);
+		if (buf == NULL) {
+			shell_error(sh, "Unable to allocate HCI buffer");
+			return -ENOMEM;
+		}
 		net_buf_add_mem(buf, hex_data, len);
```

Comment:
```suggestion
		}
		
		net_buf_add_mem(buf, hex_data, len);
```

Because you usually request that as well ;)

Reply:
> Because you usually request that as well ;)

That was actually a conscious choice after going back-and-fort a bit. I can imagine arguments both ways, e.g. whitespace should be used to group "things belonging together" and these 6 lines "belong together". That said, the encoding into the buffer could very well be seen as a separate logical chunk, so I'll add the empty line :)